index.html

```diff
@@ -0,0 +1,209 @@
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!------------------------------HEADER END------------------------------------>
+    <!------------------------------MAIN START------------------------------------>
+    <main>
+      <section class="main-opening container">
+        <p class="intro">Bringing Total healthcare to the community</p>
+        <h1 class="topic">
+          Adopting Global Best Practices in
+          <br />
+          Community Pharmacy Practice
+        </h1>
+        <p class="description">
+          Attended by most major manufacturers, distributors and community
+          Pharmcists in the Pharmacy space, the ACPN conference provides a
+          focused and unique opportunity to learn, share best practices, network
+          with peers, and interact with other healthcare providers in the
+          pharmaceutical space. At the ACPN Conference, community Pharmacists
+          can learn how best to offer pharmaceutical care and gain insights on
+          more efficient drug therapy management.
+        </p>
+        <time datetime="2023-05-20T19:30">June 15, 2023 @ 9:00am</time>
+        <p class="venue">
+          Festival Hotel, along Amuwo Odofin, Festac town, Lagos State, Nigeria.
+        </p>
+      </section>
+    </main>
+    <!------------------------------MAIN  END------------------------------------->
+
+    <!------------------------------PROGRAMS STARTS------------------------------->
+    <section class="events">
+      <div class="container">
+        <h2>Main Program</h2>
+        <hr />
+        <div class="events-content">
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-analysis"></i>
+              <h3 class="word">Exhibition</h3>
+            </div>
+            <p class="para para1">
+              Pharma Companies will be exhibiting their products and services.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-tv-retro"></i>
+              <h3 class="word">Excursion</h3>
+            </div>
+            <p class="para">
+              Explore the vast Historic collections at the national Museum.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3 class="word">Workshop</h3>
+            </div>
+            <p class="para">
+              There will be Vaccination training for Pharmacists.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-presentation-edit"></i>
+              <h3 class="word">Novelty Match</h3>
+            </div>
+            <p class="para">
+              There will be a Novelty Match between the different state arm of
+              the association.
+            </p>
```

Comment:
- [ ] Good job adding the margins and spacing to the main section as suggested by the previous reviewer. :+1: :clap:  But I think there are still some improvements you need to make here before moving to the next milestone.
- As part of the project requirement, Your project is meant to look good in all major screen sizes, the main program section does not align well on mobile screens of `419px` to `767px` Please you need to fix this :+1: :smile: 
SCREENSHOT:
![Screenshot from 2023-04-25 18-15-31](https://user-images.githubusercontent.com/107632780/234354615-76bc07b0-f1e7-4244-ac8e-876f21a49e98.png)

FIGMA DESIGN:
![Screenshot from 2023-04-25 18-17-39](https://user-images.githubusercontent.com/107632780/234354667-e075633b-8de4-478f-8b11-360748434d63.png)

- [ ] Also please make sure that the button spacing and margin match that of the provided template. :1st_place_medal:

Reply:
This is not clear to me @V-Blaze